backend/models/form.py

```diff
@@ -40,67 +43,68 @@ class Form(BaseModel):
     questions: list[Question]
     name: str
     description: str
-    submitted_text: t.Optional[str] = None
+    submitted_text: str | None = None
     webhook: _WebHook = None
-    discord_role: t.Optional[str]
-    response_readers: t.Optional[list[str]]
-    editors: t.Optional[list[str]]
+    discord_role: str | None
+    response_readers: list[str] | None
+    editors: list[str] | None
 
     class Config:
         allow_population_by_field_name = True
 
     @validator("features")
-    def validate_features(cls, value: list[str]) -> t.Optional[list[str]]:
+    def validate_features(cls, value: list[str]) -> list[str] | None:
         """Validates is all features in allowed list."""
         # Uppercase everything to avoid mixed case in DB
         value = [v.upper() for v in value]
         allowed_values = [v.value for v in FormFeatures.__members__.values()]
         if any(v not in allowed_values for v in value):
-            raise ValueError("Form features list contains one or more invalid values.")
+            msg = "Form features list contains one or more invalid values."
+            raise ValueError(msg)
 
         if FormFeatures.REQUIRES_LOGIN.value not in value:
             if FormFeatures.COLLECT_EMAIL.value in value:
+                msg = "COLLECT_EMAIL feature require REQUIRES_LOGIN feature."
                 raise ValueError(
-                    "COLLECT_EMAIL feature require REQUIRES_LOGIN feature."
+                    msg,
                 )
 
             if FormFeatures.ASSIGN_ROLE.value in value:
-                raise ValueError("ASSIGN_ROLE feature require REQUIRES_LOGIN feature.")
+                msg = "ASSIGN_ROLE feature require REQUIRES_LOGIN feature."
+                raise ValueError(msg)
 
         return value
 
     @validator("response_readers", "editors")
-    def validate_role_scoping(cls, value: t.Optional[list[str]]) -> t.Optional[list[str]]:
+    def validate_role_scoping(cls, value: list[str] | None) -> list[str] | None:
         """Ensure special role based permissions aren't granted to the @everyone role."""
-        if value and str(DISCORD_GUILD) in value:
-            raise ValueError("You can not add the everyone role as an access scope.")
+        if value and DISCORD_GUILD in value:
+            msg = "You can not add the everyone role as an access scope."
+            raise ValueError(msg)
         return value
 
     @root_validator
-    def validate_role(cls, values: dict[str, t.Any]) -> t.Optional[dict[str, t.Any]]:
+    def validate_role(cls, values: dict[str, t.Any]) -> dict[str, t.Any] | None:
         """Validates does Discord role provided when flag provided."""
-        if (
-            FormFeatures.ASSIGN_ROLE.value in values.get("features", [])
-            and not values.get("discord_role")
+        if FormFeatures.ASSIGN_ROLE.value in values.get("features", []) and not values.get(
+            "discord_role"
         ):
```

Comment:
I find the previous format more readable. I suppose the best we can do now is to assign the left operand of  `and` to a variable before the `if`.